packages/ui/src/working-groups/hooks/useWorkingGroup.ts

```diff
@@ -1,10 +1,26 @@
-import { useGetWorkingGroupQuery } from '../queries'
+import { useGetWorkingGroupOpeningsConnectionQuery, useGetWorkingGroupQuery } from '../queries'
 import { asWorkingGroup } from '../types'
 
 interface WhereInput {
   name?: string | undefined
 }
 
+// export function useCountWorkers(groupId: string) {
+//   const { data, loading } = useGetWorkersConnectionQuery({
+//     variables: { groupId, onlyActive: true },
+//   })
+//
+//   return loading ? 0 : data?.workersConnection.totalCount
+// }
```

Comment:
:police\_car:
```suggestion
```

Reply:
Will the test for WorkingGroupListItem be OK?